x-pack/plugin/sql/sql-client/src/test/java/org/elasticsearch/xpack/sql/client/UriUtilsTests.java

```diff
@@ -59,15 +82,197 @@ public void testHttpQuery() throws Exception {
 
     public void testUnsupportedProtocol() throws Exception {
         assertEquals(
-                "Invalid connection configuration [ftp://server:9201/]: Only http and https protocols are supported",
+                "Invalid connection scheme [ftp] configuration: only http and https protocols are supported",
                 expectThrows(IllegalArgumentException.class, () -> parseURI("ftp://server:9201/", DEFAULT_URI)).getMessage()
         );
     }
 
-    public void testMalformed() throws Exception {
+    public void testMalformedWhiteSpace() throws Exception {
         assertEquals(
-                "Invalid connection configuration []: Expected authority at index 7: http://",
-                expectThrows(IllegalArgumentException.class, () -> parseURI("", DEFAULT_URI)).getMessage()
+            "Invalid connection configuration: Illegal character in authority at index 7: http:// ",
+            expectThrows(IllegalArgumentException.class, () -> parseURI(" ", DEFAULT_URI)).getMessage()
+        );
+    }
+
+    public void testNoRedaction() {
+        assertEquals(
+                "Invalid connection configuration: Illegal character in fragment at index 16: HTTP://host#frag#ment",
```

Comment:
Minor: Those 2 lines seem to have inconsistent identation compared to the rest of the file.

Reply:
Ah, rite. Thanks!
This file had a bit of a styles mix. I've aligned it to what seems to have been an initial one, but happy to re-adjust it if I got it wrong.